app/javascript/widget/composables/useNextAvailability.js

```diff
@@ -0,0 +1,281 @@
+import { ref, computed } from 'vue';
```

Comment:
@fayazara Can you write the specs for the the composition `useNextAvailability.js` or is all the cases already covered in `useAvailability.js`?